source/extensions/filters/http/aws_request_signing/aws_request_signing_filter.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include "envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.pb.h"
+#include "envoy/http/filter.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
+
+#include "extensions/filters/http/common/aws/signer.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AWSRequestSigningFilter {
```

Comment:
nit, per Google style guide, this should be `AwsRequestSigningFilter` (and `Aws` everywhere else for class names)
> _For the purposes of the naming rules below, a "word" is anything that you would write in English without internal spaces. This includes abbreviations and acronyms; e.g., for "camel case" or "Pascal case," in which the first letter of each word is capitalized, use a name like StartRpc(), not StartRPC()._

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

i.e. we have `HttpFilters::Common::Aws::Signer` on line 51, not `HTTPFilters::Common::AWS::Signer`